theme/static/js/custom.js

```diff
@@ -171,26 +171,43 @@ $(document).ready(function () {
         return false
     });
 
-    $.ajax({
-        url: "/hsapi/userInfo/",
-        // success: function(user) {
-        //     if(!user.organization) {
-        //         // Disable publishing resources
-        //         if ($("#publish").length) {
-        //             $("#publish").toggleClass("disabled", true);
-        //             $("#publish").removeAttr("data-toggle");   // Disable the agreement modal
-        //             $("#publish > [data-toggle='tooltip']").attr("data-original-title",
-        //                 "Your profile information must be complete before you can formally publish resources.");
-        //         }
-        //
-        //         var message = 'Your profile is nearly complete. Please fill in the '
-        //             + '<strong>Organization</strong> field'
-        //             + ' on the <a href="/user/' + user.id + '/">User Profile</a> page';
-        //         showUniversalMessage("warn", message, 10000)();
-        //     }
-        // },
-        error: showUniversalMessage()
-    })
+    if (window.location.hash == '#_'){
+        $.ajax({
+            url: "/hsapi/userInfo/",
+            success: function(user) {
+                console.log(user)
+
+                var missing_fields = []
```

Comment:
I'd go with using semicolons consistently, since that's what the rest of the code appears to do.

Reply:
They also appear to use the space between `function` and `()` convention, so I'd do that too...and they have a space between the keywords and brackets (so `if (blah) {` rather than `if(blah){`.